fpga/src/main/resources/vcu118/sdboot/sd.c

```diff
@@ -168,9 +170,11 @@ static int copy(void)
 	int rc = 0;
 
 	dputs("CMD18");
+
+	kprintf("LOADING 0x%xB PAYLOAD\r\n", PAYLOAD_SIZE_B);
 	kprintf("LOADING  ");
 
-    // John: Let's go slow until we get this working
+	// TODO: Speed up SPI freq. (breaks between these two values)
```

Comment:
I would either file an issue to look at this in the future or just delete this comment and the other line.

Reply:
https://github.com/ucb-bar/chipyard/issues/854